src/Generating/StructGen.h

```diff
@@ -59,7 +60,10 @@ class cStructGenTrees :
 		sSetBlockVector & a_Overflow
 	);
 
-	int GetNumTrees(
+	/** Get the number of Tree to generate for chunk (a_ChunkX; a_ChunkZ)
+	if it returns zero, a_randomNum is set to a range of 0 to 1023 indicating the probability that a tree should spawn: a_RandomNum / cStructGenTrees::DividerForOnePerC
+	else a_RandomNum is 0 */
+	double GetNumTrees(
```

Comment:
This doxy-comment makes absolutely no sense. Please clarify it.

Reply:
Sorry I forgot to change it in an preview commit